JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -402,14 +402,18 @@ - (void)scrollToIndexPath:(NSIndexPath *)indexPath animated:(BOOL)animated
     }
 
     CGFloat collectionViewContentHeight = [self.collectionView.collectionViewLayout collectionViewContentSize].height;
-    BOOL isContentTooSmall = (collectionViewContentHeight < CGRectGetHeight(self.collectionView.bounds));
-
+    CGFloat collectionViewHeight = CGRectGetHeight(self.collectionView.bounds);
+    BOOL isContentTooSmall = (collectionViewContentHeight < collectionViewHeight + 10.f);
```

Comment:
Yeah, probably should have give some commented explanation. 

Not sure why this is needed, but if `isContentTooSmall` is too small like this then we'll see UIKit "not-scrolling" bug. And not sure what exact number we needed - probably because of unknown nature of this UI bug